examples/ecn_alan.ipynb

```diff
@@ -0,0 +1,199 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "## Building an ECN Eval\n",
+    "\n",
+    "This notebook shows how to build and run an eval following instructions in docs/build-eval.md"
```

Comment:
tinyt: I really like that you shared the inspiration in the PR OP, wondering if it's worth putting it here too.

Reply:
1ab986a